src/uu/tee/src/tee.rs

```diff
@@ -209,6 +194,6 @@ impl Read for NamedReader {
 }
 
 fn warn(message: &str) -> Error {
-    eprintln!("{}: {}", NAME, message);
-    Error::new(ErrorKind::Other, format!("{}: {}", NAME, message))
+    show_warning!("{}", message);
```

Comment:
I was talking about replacing call to warn() by show_warning() 
and remove this function from this file :)

Reply:
Okay, but what about the `Error::new` ? That would be missing if warn() gets replaced.
Also do happen to know how I could provoke an error to test the output?

Reply:
done